models/address.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import "k8s.io/api/core/v1"
+
+type Addresses []Address
+type Address struct {
+	Kind string `json:"kind"`
+	Name string `json:"name"`
+	IP   string `json:"ip"`
+}
+
+func (addresses *Addresses) Parse(as []v1.EndpointAddress) {
```

Comment:
In next PR we can merge Parse method and pass by argument interface{} and check if it's an array inside and type. But for now LGFM to continue working.

Example:
``go
func name(v interface{}) {
    switch x := v.(type) {
    case []MyInterface:
        fmt.Println("[]MyInterface, len:", len(x))
        for _, i := range x {
            fmt.Println(i)
        }
    case MyInterface:
        fmt.Println("MyInterface:", x)
    default:
        fmt.Printf("Unsupported type: %T\n", x)
    }
```

Reply:
Is that a best practice in Go? For me it sound like put more logic in one function instead of distribute it in multiple ones.

Reply:
I think is more a point of view.